client/src/components/Map/components/Block.js

```diff
@@ -129,7 +129,7 @@ export class Block extends Component {
     });
     return (
       <li className={`block ${isExpanded ? 'open' : ''}`}>
-        <button
+        <div
```

Comment:
Why changed to `div`?

Reply:
I will revert that. thank you for noticing it.